src/typings-new-format/spec-1.0/methods.ts

```diff
@@ -89,14 +113,10 @@ export interface MethodBase10 extends Operation10 {
      * A method's body is defined in the body property as a hashmap, in which
      * the key MUST be a valid media type.
      */
-    body?: { [key: string]: datamodel.TypeReference10 }
+    body?:  datamodel.TypeReference10[]
```

Comment:
Actually, body is always a type declaration with name equal to the specified media type. If user specifies some type for body, it goes to supertypes (i.e. 'type' array). For example,
```
body:
  application/json: lib.MyType
```
now becomes
```
body: {
  "application/json" : {
    "name": "application/json",
    "type": [ "lib.MyType" ]
  }
}
```
and in the new format it will be 
```
body: [
  {
    "name": "application/json",
    "type": [ "lib.MyType" ]
  }
]
```